### DISCUSSION TITLE
Transfer balance to bank account?

### DISCUSSION THREAD

**Discussion by dzg** [mmohades/Venmo#75] [General]
This is awesome!

But I'm trying to write a script that will transfer my Venmo balance to my bank account.

Not seeing how to (a) get my balance and (b) transfer to bank.

Is it possible?

Thanks!

**Comment by mmohades** [comment]
It should be possible but I haven't looked into it. Have you been able to do it?

**Comment by VasuBhog** [comment]
It seems that get_user_transactions returns back Transaction class and therefore it only returns `Payment` transactions instead of `Transfer`.
<img width="524" alt="Screen Shot 2022-07-26 at 1 41 34 PM" src="https://user-images.githubusercontent.com/23587151/181108510-4087d943-c63e-41df-b8cc-93e6c193cea9.png">

https://github.com/mmohades/Venmo/blob/master/venmo_api/models/transaction.py#L66

Is there a way to change the transaction type or am I missing something? @mmohades

**Reply by mmohades** [reply]
I see. Yeah I think I'd need to add that type. There are some other types of transactions that aren't currently supported like the one made by the debit card. These are skipped because they have different schemas or I haven't seen such transactions on my account.
